app/src/main/java/com/example/mvp9/data/UserRepostitory.java

```diff
@@ -0,0 +1,34 @@
+package com.example.mvp9.data;
+
+import com.example.mvp9.data.local.UserDataSource;
+import com.example.mvp9.model.User;
+
+public class UserRepostitory implements UserDataSource {
+    private UserDataSource mUserDataLocal;
+    private static  UserRepostitory instance;
```

Comment:
```suggestion
    private static  UserRepostitory sInstance;
```
Chú ý đặt tên theo convention

Reply:
Đặt biến này lên trên biến private

Reply:
convention 2.8